### PR TITLE
Add priority parser

### PR DESCRIPTION
**Pull request by AmirKremer**

### CODE REVIEW

File: spec/lib/priority_parser_spec.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      expect(pickups.second.supplier.suppliers[4]).to have_attributes(name: "קפה גרג- סניף שבעת הכוכבים")
+      expect(pickups.second.supplier.suppliers[5]).to have_attributes(name: "אוניון- סניף שבעת הכוכבים")
+      expect(pickups.second.supplier.suppliers[6]).to have_attributes(name: "סטריט מיט- סניף שבעת הכוכבים")
+      expect(pickups.second.supplier.suppliers[7]).to have_attributes(name: "ארומה- סניף שבעת הכוכבים")
+      expect(pickups.second.supplier.suppliers[8]).to have_attributes(name: "שוק אוכל קניום שבעת הכוכבים")
+      expect(pickups.second.supplier.suppliers[9]).to have_attributes(name: "השניצליה - סניף שבעת הכוכבים")
+      expect(pickups.second.supplier.suppliers[10]).to have_attributes(name: "פיצה פינו -קניון שבעת הכוכבים")
+    end
+
+    it "supports processing twice" do
```
Comment (eluxen) [inline spec/lib/priority_parser_spec.rb]: how about it 'overrides existing pickups' ?

File: lib/priority_parser.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      user_name: 3,
+      user_phone: 4,
+      user_email: 5,
+      supplier_id: 7,
+      supplier_name: 8,
+      status: 9,
+      sub_suppliers: 20.times.map { |i| { id: i * 2 + 10, name: i * 2 + 11 } }
+  }
+
+  def self.process(pickups)
```
Comment (eluxen) [inline lib/priority_parser.rb]: General note -
wdyt about error handling? should we fail completely if there is a problem with a specific pickup?
  
we should also consider logging the number of pickups we processed, just to make sure that we got it right.

**Comment by eluxen** [comment]
Approve!

**Comment by eluxen** [comment]
merge it! I need data :)

File: lib/priority_parser.rb (hunk 2 above)
Comment (AmirKremer) [inline lib/priority_parser.rb]: Let's create another ticket for that :)

**Pull request merged**